kitsune/questions/config.py

```diff
@@ -244,11 +244,11 @@
                 "categories": OrderedDict(
                     [
                         (
-                            "install-and-update-firefox-ios",
+                            "install-and-update",
                             {
                                 "name": _lazy("Install and Update"),
```

Comment:
I think this should match the actual topic name, so the "update" shouldn't be capitalized?
```suggestion
                                "name": _lazy("Install and update"),
```